vissl/hooks/log_hooks.py

```diff
@@ -207,7 +217,10 @@ def on_update(self, task: "tasks.ClassyTask") -> None:
                         f"btime({self.btime_freq}iters): {rolling_btime} ms; "
                         f"rolling_eta: {rolling_eta_str}"
                     )
+                    log_data["btime_iters"] = rolling_btime
+                    log_data["rolling_eta"] = rolling_eta_str
                 logging.info(log_str)
+                save_file(log_data, f"{task.checkpoint_folder}/stdout.json")
```

Comment:
I just realized that the call to `save_file` will consistently spew the logs https://github.com/MLH-Fellowship/vissl/blob/master/vissl/utils/io.py#L56 which will be noisy. Yet we can't disable the logging in L56 in `save_file`. I recommend to replace the `save_file(...)` with 
```
with PathManager.open(f"{task.checkpoint_folder}/stdout.json", "a") as fopen:
        fopen.write(json.dumps(data, sort_keys=False) + "\n")
        fopen.flush()
```